internal/data/receivers.go

```diff
@@ -460,6 +460,33 @@ func (r *ReceiverModel) GetByContacts(ctx context.Context, sqlExec db.SQLExecute
 	return receivers, nil
 }
 
+// GetByWalletAddress search for receivers by wallet address.
+func (r *ReceiverModel) GetByWalletAddress(ctx context.Context, sqlExec db.SQLExecuter, walletAddresses ...string) ([]*Receiver, error) {
```

Comment:
```suggestion
// GetByReceiverWalletAddress search for receivers by wallet address.
func (r *ReceiverModel) GetByWalletAddress(ctx context.Context, sqlExec db.SQLExecuter, receiverWalletAddresses ...string) ([]*Receiver, error) {
```

For desambiguation, we should call it receiverWallet, since we have different concepts in our application and database for wallet and receiverWallet.

Reply:
addressed